### ISSUE TITLE
Ask for training sequence_num

### ISSUE THREAD

**Issue by Siyuan-Liu99** [OamIcs/MoSim#4]
Hi, happy to see this nice world model.
I have a little question about the dataset sequence_num and step_num for myohand, I find the test dataset were collect as follow:
`collect_data_myo(env, step_num=1000, random_ex=5, static_prop=0.1, 
                     save_path=save_path, data_name='myohand_test_motor', 
                     env_name='myoHandPoseFixed-v0', sequence_num=300, save_images=False)`
I wonder to kown how long sequence were used for training to get the myohand world model, 
since I didn't find the dataset for myohand on your hugging face. 
Looking forward to your help, thanks again.

**Comment by Siyuan-Liu99** [comment]
Is that the train sequence_num for myohand is 10000? Since i find other training dataset is always ten times test dateset.

**Comment by wuji-ss1** [comment]
I think we should set the training dataset size to 3,000, since the test set uses sequence_num = 300. Also, if you want to simulate MyoSim, remember to switch to motor control mode, because our model assumes the environment is Markovian. Additionally, you may want to set skip_frame = 1 in the environment parameters. These modifications are already included in the myo_sim.zip file in the repository, so you can simply follow the instructions there to replace the files.

**Comment by Siyuan-Liu99** [comment]
yes, I have follow the instructions to replace the files, thanks for your help, I will try to simulate MyoSim
